### ISSUE TITLE
[Backgound Media]: QA: Screen reader user is not aware of exact use of video content

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9711]
### Description

Page Element:
After activation of the play button

Issue Description:
A video content without audio is played when we activate the play button. Hence the user who is blind may not be aware of it and wonder what happens after activating the play button. 

Recommend Fix:
Provide the following message in aria-describedby and associate with the play button. 
Note: This plays A video only content

User Impact:
When the screen reader doesn’t identify the page elements, it impacts the experience of users who are blind or have vision impairment. They can miss important information or fail to perform an action on the web page. 

Tool:
JAWS_Desktop
Voiceover_iPhone

### Component(s) impacted

Backgound Media

### Browser

Safari, Firefox

### Carbon for IBM.com version

v1.40.0-rc.0

### Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components-react

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
